modules/orchestrator/con-understanding-orchestrator-plugin-dependencies.adoc

```diff
@@ -0,0 +1,48 @@
+:_mod-docs-content-type: CONCEPT
+[id="con-understanding-orchestrator-plugin-dependencies_{context}"]
+= Understanding Orchestrator plugin dependencies
+
+The Orchestrator plugin in {product-very-short} requires components such as SonataFlow and supporting network policies to run. When you enable the plugin in your {product-custom-resource-type} custom resource (CR), the Operator automatically provisions these dependencies for you:
```

Comment:
"Orchestrator Plugin dependencies" is something specific to the RHDH Operator. Maybe we could clarify that in the title? Maybe it is just me, but as it is currently structured, it seems like it is describing how to understand orchestrator plugin dependencies regardless of the install method (since you merged the content from the Helm Chart as well).